Dockerfile.template

```diff
@@ -85,10 +86,28 @@ RUN sudo apt-get update && sudo apt-get install -y \
 	php -m | grep "gd" && \
 	php -m | grep "sockets"
 
+
+	# Copies the docker-php-ext helper scripts and allows them to be used in the image
+COPY /docker-php-scripts/docker-php-* /usr/local/bin/
+COPY /docker-php-scripts/install-php-extensions /usr/local/bin/	
+
+RUN cd /usr/local/bin && \
+	sudo chmod +x /usr/local/bin/install-php-extensions /usr/local/bin/docker-php-ext-configure /usr/local/bin/docker-php-ext-enable /usr/local/bin/docker-php-ext-install /usr/local/bin/docker-php-source
+
+ENV PHP_INI_DIR /usr/local/etc/php
+
+RUN sudo mkdir -p "$PHP_INI_DIR/conf.d"; \
+	# allow running as an arbitrary user (https://github.com/docker-library/php/issues/743)
+	[ ! -d /var/www/html ]; \
+	sudo mkdir -p /var/www/html; \
+	sudo chown www-data:www-data /var/www/html; \
+	sudo chmod 777 /var/www/html
```

Comment:
1. These 4 lines can be simplified.
2. For Dockerfiles, we prefer to use `&& \` instead of `; \` so that if a command fails, we stop instead of keep going.

So instead of:

```bash
ENV PHP_INI_DIR /usr/local/etc/php

RUN sudo mkdir -p "$PHP_INI_DIR/conf.d"; \
	# allow running as an arbitrary user (https://github.com/docker-library/php/issues/743)
	[ ! -d /var/www/html ]; \
	sudo mkdir -p /var/www/html; \
	sudo chown www-data:www-data /var/www/html; \
	sudo chmod 777 /var/www/html
```

we can do:

```bash
ENV PHP_INI_DIR /usr/local/etc/php
RUN sudo mkdir -p "$PHP_INI_DIR/conf.d" && \
	# allow running as an arbitrary user (https://github.com/docker-library/php/issues/743)
	sudo mkdir -p /var/www/html && \
	sudo chown www-data:www-data /var/www/html && \
	sudo chmod 777 /var/www/html
```

but also, we are we setting up a web server directory in this PR?